src/Footer/Footer.js

```diff
@@ -29,7 +50,10 @@ const Footer = ({ resetDetails, form }) => {
             id="deny-access"
             type="submit"
             buttonStyle="danger mega"
-            onClick={() => {}}
+            onClick={(e) => {
+              e.preventDefault();
+              handleAccessLog(DENIED);
+            }}
```

Comment:
Could you create a separate function out of `return` statement as we discussed before?